exo-go/src/service_config_helpers/index.go

```diff
@@ -0,0 +1,26 @@
+package serviceConfigHelpers
+
+import (
+	"io/ioutil"
+	"log"
+	"path"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// GetServiceConfig reads service.yml of the role service and returns
+// the serviceConfig object
+func GetServiceConfig(location string) types.ServiceConfig {
```

Comment:
`location` refers to the name of the service folder. It aligns with the [key](https://github.com/Originate/exosphere/blob/master/exosphere-shared/example-apps/running/application.yml#L12) in `application.yml`. It's not really a servicePath since we do `path.Join(location, "service.yml")` within the function.